autoload/lsp/omni.vim

```diff
@@ -201,7 +207,35 @@ function! lsp#omni#get_vim_completion_item(item, ...) abort
         endif
     endif
 
+    if exists('l:snippet')
+        let l:completion['user_data'] = string([l:trigger, l:snippet])
```

Comment:
`user_data` is only available in certain version of vim. We need to check if vim has that patch.

Reply:
In `plugin/lsp.vim`, I already check if Vim `has('patch-8.0.1493')`. Would you prefer I set `g:lsp_ultisnips_integration` back to `0` there and/or add an additional `has('patch-8.0.1493')` to `if exists('l:snippet')`?